### PR TITLE
Proofing

### PR DESCRIPTION
**Pull request by hallaathrad**
This includes spelling and tense tuning. Added some
internationalisation.
I took the liberty of rephrasing some things to keep the tone
conversational yet clear.

Left some comments with doubts or questions.
I suggest also reviewing the form for speaker sign-up.

---

Dec 6: Speaker sign-up and Sponsor call for proposals form have been updated.

### CODE REVIEW

File: speak/index.html [hunk 1]
```diff
@@ -4,76 +4,76 @@
...
-	<p>We are commited to increasing the diversity of our event and all tech events are offer a 10% bonus to the base compensation for those coming from typically underrepresented demographics.</p>
+	<p>All speakers at #fstoco will be paid a base honorarium of $1000CAD, we will then reimburse you for travel and lodging expenses in Canadian Dollars up to a certain amount based on your geographical region. Reimbursement will only happen if you book through our travel and lodging partners, while some few exceptions can be made for those unable to book with them. You must provide all receipts within 30 days of the end of the event in order to be reimbursed.</p>
+	<p>We are fully invested in increasing the diversity of our event and all tech events are offered a 10% bonus to the base compensation for those coming from typically underrepresented demographics.</p>
```
Comment (fullstackto) [inline speak/index.html]: This line doesn't make sense

File: index.html [hunk 2]
```diff
@@ -4,40 +4,48 @@
 ---
 <section>
 	<h2>About #fstoco</h2>
-	<p>#fstoco is put on by Full Stack Toronto a non-profit geared towards providing web professionals with intermediate and advanced educational resources. #fstoco is a unique conference because it focuses on all layers of the web: front-end, back-end, devops, ux, design, project management, and product development.</p>
+	<p>#fstoco is put on by Full Stack Toronto, a non-profit organisation geared towards providing web professionals with intermediate and advanced educational resources. #fstoco is a unique conference because it deals with all the layers of the web: Front-end, Back-end, DevOps, UX and visual designs, project management, and product development.</p>
```
Comment (fullstackto) [inline index.html]: "deals with" is to problem specific, i'd prefer focuses on, as thats on our branding

Comment (hallaathrad) [inline index.html]: The issue I saw is that "focus" implies narrowing from many to single. We're talking about the full stack, which means to my understanding the broad spectrum: exactly the opposite of focusing. 
However, the final call is yours and I'll change it to focus if you feel it's more appropriate.

File: speak/index.html (hunk 1 above)
Comment (hallaathrad) [inline speak/index.html]: I saw "committed" too many times so I chose a synonym. 
if  not "fully invested" or "committed", maybe devoted or dedicated?

Comment (fullstackto) [inline speak/index.html]: No this part " the diversity of our event and all tech events are offered a 10% b"

Comment (hallaathrad) [inline speak/index.html]: Oh. That one was actually like that in the original and I thought it was some sort of technical expression I didn't quite understand myself... LOL

How about we change that line for something like:
"We are fully invested in increasing the diversity of our event, so we grant a 10% bonus over the base compensation to every speaker coming from typically underrepresented demographics."

File: index.html (hunk 2 above)
Comment (fullstackto) [inline index.html]: we just need something softer than "deals"

File: speak/index.html (hunk 1 above)
Comment (fullstackto) [inline speak/index.html]: "We are fully invested in increasing the diversity at our events, we grant a 10% bonus over the base compensation to every speaker coming from typically underrepresented demographics."

File: index.html (hunk 2 above)
Comment (hallaathrad) [inline index.html]: Covers or explores?

Comment (fullstackto) [inline index.html]: yah "covers all the layers"

**Pull request merged**